config/routing.go

```diff
@@ -5,5 +5,44 @@ type Routing struct {
 	// Type sets default daemon routing mode.
 	//
 	// Can be one of "dht", "dhtclient", "dhtserver", "none", or unset.
-	Type string
+	Type string `json:",omitempty"`
```

Comment:
@ajnavarro  `*OptionalString` will be better here: it will allow us to control the implicit default when missing (see how we do it for  `UnixFSShardingSizeThreshold`  in `./core/node/groups.go`):


```suggestion
	Type *OptionalString  `json:",omitempty"`
```

Reply:
changed to OptionalString